docs/allowedwords.md

```diff
@@ -0,0 +1,21 @@
+## How to add allowed word
+
+All the allowed words are located in an array in [src/config.ts](src/config.ts) inside the object ` ALEX ` as shown below. 
+
+```ts
+ALEX: {
+    profanitySureness: 2,
+    noBinary: true,
+    allow: [
+      'just',
+      'brother-sister',
+      'brothers-sisters',
+      'daft',
+      ...
+
+```
```

Comment:
I think there might be some sintax missing here, or is this intended?
```suggestion
```ts
ALEX: {
    profanitySureness: 2,
    noBinary: true,
    allow: [
      'just',
      'brother-sister',
      'brothers-sisters',
      'daft',
      ...
    ]
}
```
```

Reply:
This is how it is currently in the readme. We can change it to this 👍